mteb/overview.py

```diff
@@ -350,6 +364,8 @@ def get_tasks(
         _tasks = filter_tasks_by_modalities(
             _tasks, modalities, exclusive_modality_filter
         )
+    if exclude_aggregate:
+        _tasks = filter_aggregate_tasks(_tasks, exclude_aggregate)
```

Comment:
Follow up on the comment above, we can omit passing the flag in.
```suggestion
    if exclude_aggregate:
        _tasks = filter_aggregate_tasks(_tasks)
```